article-list.html

```diff
@@ -14,33 +14,54 @@
             <!-- This div is for the list of articles on the page -->
             <div class="article-list">
                 <h1>All Articles</h1>
-				<div class="image">
-					<p>This will be an image</p>
+				<div class="flex-container">

```

Comment:
In this version, I completely changed the layout and used a flexbox to display each article